src/status_im/contexts/profile/config.cljs

```diff
@@ -9,21 +9,26 @@
 (defn login
   []
   {;; Temporary fix until https://github.com/status-im/status-go/issues/3024 is resolved
-   :wakuV2Nameserver            "8.8.8.8"
-   :openseaAPIKey               config/opensea-api-key
-   :poktToken                   config/POKT_TOKEN
-   :infuraToken                 config/INFURA_TOKEN
-   :raribleMainnetAPIKey        config/RARIBLE_MAINNET_API_KEY
-   :raribleTestnetAPIKey        config/RARIBLE_TESTNET_API_KEY
-   :alchemyEthereumMainnetToken config/ALCHEMY_ETHEREUM_MAINNET_TOKEN
-   :alchemyEthereumGoerliToken  config/ALCHEMY_ETHEREUM_GOERLI_TOKEN
-   :alchemyEthereumSepoliaToken config/ALCHEMY_ETHEREUM_SEPOLIA_TOKEN
-   :alchemyOptimismMainnetToken config/ALCHEMY_OPTIMISM_MAINNET_TOKEN
-   :alchemyOptimismGoerliToken  config/ALCHEMY_OPTIMISM_GOERLI_TOKEN
-   :alchemyOptimismSepoliaToken config/ALCHEMY_OPTIMISM_SEPOLIA_TOKEN
-   :alchemyArbitrumMainnetToken config/ALCHEMY_ARBITRUM_MAINNET_TOKEN
-   :alchemyArbitrumGoerliToken  config/ALCHEMY_ARBITRUM_GOERLI_TOKEN
-   :alchemyArbitrumSepoliaToken config/ALCHEMY_ARBITRUM_SEPOLIA_TOKEN})
+   :wakuV2Nameserver              "8.8.8.8"
+   :statusProxyEnabled            config/status-proxy-enabled?
+   :statusProxyMarketUser         config/STATUS_BUILD_PROXY_USER
```

Comment:
In this file, wouldn't it be better to use different env vars for `:statusProxyMarketUser` and `:statusProxyMarketPassword` instead of reusing `STATUS_BUILD_PROXY_USER` and `STATUS_BUILD_PROXY_PASSWORD`, respectfully? If we want to reuse values, we can do inside the namespace `status-im.config`, but to the consumers of `status-im.config`, they wouldn't know. Seems a bit more future proof. This seems to be what status-desktop is doing https://github.com/status-im/status-desktop/pull/15913/files#diff-92c30cc52000ad1754ec5845cb656b2d8fe5a050db412429093cbdd9bb83d2e1R222.

Reply:
As you pointed out to me in private my suggestion doesn't work, we will get this error when compiling to JS:

```
  22 | (goog-define STATUS_BUILD_PROXY_MARKET_USER STATUS_BUILD_PROXY_USER)
-------^------------------------------------------------------------------------
Encountered error when macroexpanding cljs.core/goog-define.
```

Another solution is to just use different vars, but not reuse them with each other, this seems better. My reasoning is that the app code shouldn't know that `STATUS_BUILD_PROXY_MARKET_USER` is the same as `STATUS_BUILD_PROXY_USER`. It will be up to the build process to pass the correct values and decide.

If you prefer to go back to the original solution that is good for now too. Will approve the PR in advance 👍🏼